packages/substrate-client/src/chainhead/chainhead.ts

```diff
@@ -16,46 +17,43 @@ import {
 } from "@/utils/subscriptions-manager"
 
 function isOpeartionEvent(
```

Comment:
```suggestion
function isOperationEvent(
```

Reply:
it took me at least 10 secs to notice the diff 😂 Thanks!